### PR TITLE
Bookstore Milestone: Initialize project with components

### PR DESCRIPTION
**Pull request by CarlosIgreda**
For this Milestone I:

✅ Used GitFlow
✅ Used linters
✅ Set up a React App
✅ Used React components
✅ Used React props
✅ Used React state
✅ Handled events in a React app
✅Used React Router

### CODE REVIEW

File: src/components/List.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  return (
+    <>
+      {books.map((book) => (
+        <article key={book.id}>
+          <h2>{book.title}</h2>
+          <p>{book.author}</p>
+          <button type="button" onClick={() => handleDelete(book.id)}>
+            Delete
+          </button>
+        </article>
```
Comment (victorokoroji) [inline src/components/List.js]: - [x] Good job so far 👍🏽. But for your components to be reusable, I will suggest you extract this code to a separate component called` <Book />`, add the necessary props and import it here.

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @CarlosIgreda, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the Git Flow ✅ 
- Nice folder structure✅ 
- The PR title and description are professional and descriptive ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/List.js (hunk 1 above)
Comment (CarlosIgreda) [reply src/components/List.js]: @vickymarz, thank you for the feedback. Just to let you know, the required change is done. I've created a new component called Book.js as you suggested.

**Review by victorokoroji — APPROVED** [review]
Hi @CarlosIgreda, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Requested change implemented ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by CarlosIgreda** [comment]
@vickymarz, thank you so much! Time to merge it 🤓.

**Pull request merged**